OtripleS.Web.Api/Services/Classrooms/ClassroomService.cs

```diff
@@ -67,5 +67,15 @@ public partial class ClassroomService : IClassroomService
 
 			return storageClassrooms;
 		});
+
+		public ValueTask<Classroom> RetrieveClassroomById(Guid classroomId) =>
+		TryCatch(async () =>
+		{
+			Classroom storageClassroom =
```

Comment:
Validation tests and some validations are missing. Please check with other entities.

Reply:
I don't why it was coming two time as i did tested and debug many time, the duplicated lines are removed.